dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/Constants.java

```diff
@@ -51,4 +51,6 @@ private Constants() {
     public static final Duration DEFAULT_TOKEN_ADDING_RATE = Duration.ofMinutes(DEFAULT_CONSUMING_TOKEN_RATE);
     public static final String FILENAME_DOCKERFILE = "dockerfile";
     public static final String FILENAME_DOCKER_COMPOSE = "docker-compose";
+    public static final String CHECK_FOR_RENOVATE = "checkforrenovate";
+    public static final String RENOVATE_CONFIG_FILEPATH = "renovate.json";
```

Comment:
While we will using `renovate.json` internally there are actually 9 different potential renovate configuration files,  see https://docs.renovatebot.com/configuration-options/ .  We might want to check for all of these paths.